content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
+```shell
+wash> ctl link MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver PORT=8080
+```
+Once you see that the link has been advertised, you are ready to send a request to your actor.
 
-```
\ No newline at end of file
+#### Interacting with your Actor
+In another terminal window (not the REPL), run the following command:
+```shell
+curl localhost:8080/echo
+```
+In response, you should receive your request object (notice the path argument):
+```shell
+{"method":"GET","path":"/echo","query_string":"","headers":{"host":"localhost:8080","user-agent":"curl/7.64.1","accept":"*/*"},"body":[]}
+```
+Feel free to try out different methods of making a request to your Actor, including adding headers or using a different HTTP method to see different outputs.
+
+Instead of using `curl`, you can also _directly invoke_ actors' registered functions using `ctl call`. As mentioned before, the function that "echoes" this HTTP request is registered with the operation `HandleRequest`, and we can make this request directly to the actor if we supply the correct parameters.
+
+Here's an example of using `call` in the REPL to mimic our above `curl`. Note that this is not interacting with the HTTP server provider, and we don't need it to be running or linked for this operation to succeed:
+```shell
+wash> ctl call MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 HandleRequest {"method": "GET", "path": "/echo", "body": "", "queryString":"","header":{}}
+```
+Our output will look something like this:
+```shell
+Call response (raw): ��statusCode�Ȧstatus�OK�header��body�S{"method":"GET","path":"/echo","query_string":"","headers":{},"body":[]}
+```
+Because of our method of serialization / deserialization, the return payload has some characters that the terminal doesn't know how to interpret from its bytes.[<sup>1</sup>](#references) However you can still see the response body which contains our exact "echoed" request.
+
+That covers the basics of wasmCloud development in our REPL environment. To learn more about Actors, Providers, and more concepts on wasmCloud, continue onto the [App Development](../../app-dev) or [Platform Building](../../platform-builder) sections depending on your role.
+
+###### References
+1. We're using [msgpack](https://msgpack.org/) to serialize arbitrary byte payloads when sending information to/from actors. Due to this serialization format, the bytes that are sent to and from actors must be deserialized with the shape of the object already known, and in the case of `ctl call` we are doing our best guess as to what the return payload is going to look like. As you can see when using `curl`, when we know the shape of the data (an HTTP response) we're able to fully deserialize the actor's response.
```

Comment:
If this is a footnote, we can actually use real markdown footnote syntax, the hugo template will even draw a little gray hard rule above your footnotes section.

Reply:
Sweet I didn't know there was an official way to do this. On it